src/qt/pivx/dashboardwidget.cpp

```diff
@@ -212,15 +211,11 @@ void DashboardWidget::loadWalletModel()
         // Notification pop-up for new transaction
         connect(txModel, &TransactionTableModel::rowsInserted, this, &DashboardWidget::processNewTransaction);
 #ifdef USE_QTCHARTS
-        // chart filter
-        stakesFilter = new TransactionFilterProxy();
-        stakesFilter->setDynamicSortFilter(true);
-        stakesFilter->setOnlyStakes(true);
-        stakesFilter->setSourceModel(txModel);
-        hasStakes = stakesFilter->rowCount() > 0;
-
-        onHideChartsChanged(walletModel->getOptionsModel()->isHideCharts());
-        connect(walletModel->getOptionsModel(), &OptionsModel::hideChartsChanged, this, &DashboardWidget::onHideChartsChanged);
+        if (!walletModel->getOptionsModel()->isHideCharts()) {
+            onHideChartsChanged(false);
+        }
```

Comment:
as was saying in the previous review, we need to keep the original line
```cpp
onHideChartsChanged(walletModel->getOptionsModel()->isHideCharts());
```

Otherwise, starting the qt wallet with `--hidecharts` doesn't "hide" the charts area, but instead it displays the image "You have no staking rewards" (since we no longer call `ui->right->setVisible(fShowCharts)`).